patches/uniffi-bindgen-react-native@0.29.3-1.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/crates/ubrn_cli/templates/jsi/android/cpp-adapter.cpp b/crates/ubrn_cli/templates/jsi/android/cpp-adapter.cpp
```

Comment:
Could you add a README.md on the same directory to document why we are patching this?